src/css-image-value.js

```diff
@@ -0,0 +1,39 @@
+(function(internal, scope) {
+
+  function CSSImageValue(image) {
+    if (!(image instanceof Image)) {
```

Comment:
Since CSSImageValue does not have a from method, you don't need to do it, no. Your CSSImageValue here also shouldn't have a from method. It should only expose methods that are in the spec.